src/adapter.js

```diff
@@ -52,6 +52,7 @@ function createQUnitStartFn (tc, runnerPassedIn) { // eslint-disable-line no-unu
             fixture.style.width = '1000px'
             fixture.style.height = '1000px'
             document.body.appendChild(fixture)
+            runner.config.fixture = fixture
```

Comment:
I believe that even if we were to use the undocumented Node type, this may not work as expected. Storing a reference to `fixture` should still cause the same bug as before because it is a live reference.

When a test adds more children to the fixture, then by the end of the text, `config.fixture` will contain those children as well, and not reset the way it should.

Perhaps set `config.fixture` to an empty string?

Reply:
The reset process runs *before* the test not *after*, also they are [cloning the node](https://github.com/qunitjs/qunit/blob/89dcd90211fb129df48aa11b72dbafd9f5a18009/runner/fixture.js#L45), so what-ever a test does should not affect the original fixture.

Empty string might work, but we also need to set the correct styles on the element. I'll check if `qunit.css` has all the necessary definitions.